source/_integrations/enocean.markdown

```diff
@@ -23,8 +23,16 @@ ha_platforms:
 
 The [EnOcean](https://en.wikipedia.org/wiki/EnOcean) standard is supported by many different vendors. There are switches and sensors of many different kinds, and typically they employ energy harvesting to get power such that no batteries are necessary.
 
-The EnOcean integration adds support for some of these devices. You will need a controller like the [USB300](https://www.enocean.com/en/enocean_modules/usb-300/) in order for it to work.
+The EnOcean integration adds support for some of these devices. You will need a controller like the [USB300](https://www.enocean.com/en/enocean_modules/usb-300/) or a EnOcean Pi868 in order for it to work.
 
+EnOcean Pi boards for Raspberry Pi need to have bluetooth disabled and uart to be enabled to be accessible via /dev/ttyAMA0.
+
+Add the following parameters to the bottom of the `/boot/config.txt` file and reboot.
+
+```text
+dtoverlay=disable-bt
+enable_uart=1
+```
```

Comment:
This is quite specific to the installation method one is running. And thus not correct for the larger part of our user base (95+%).

Reply:
I bow to your great wisdom.